tests/test_unit.py

```diff
@@ -123,6 +129,19 @@ def test__build_url(self):
         built_url = self.client._build_url(query_params)
         self.assertEqual(built_url, url)
 
+    @mock.patch('python_http_client.client.Client._make_request')
+    def test__urllib_headers(self, maker):
+        self.client._update_headers({'X-test': 'Test'})
+        self.client.get()
+        request = maker.call_args[0][1]
+        self.assertTrue('X-test' in request.headers)
```

Comment:
`self.assertIn('X-test', request.headers)` ?

* https://docs.python.org/3/library/unittest.html?highlight=assertin#unittest.TestCase.assertIn

Reply:
Available since 3.1, client provides support for 2.7 (2.6 previously)

Reply:
@3lnc 

> New in version 2.7.

* https://docs.python.org/2/library/unittest.html?highlight=assertin#unittest.TestCase.assertIn